server/pxf-hdfs/src/main/java/org/greenplum/pxf/plugins/hdfs/orc/ORCVectorizedResolver.java

```diff
@@ -127,6 +127,8 @@
 
     private List<List<OneField>> cachedBatch;
 
+    private static final String UNSUPPORTED_ERR_MESSAGE = "Current operation is not supported";
```

Comment:
maybe this would be better? or maybe even specify read/write
```suggestion
    private static final String UNSUPPORTED_ERR_MESSAGE = "Vectorized ORC operations are not supported";
```

Reply:
I am leaving the ORC related changes for now as it is since we are working on orc write and this will be changed.